calling getChatModel throw Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'update')

### Is there an existing issue for this?

- [x] I have searched the existing issues.

### Is this a problem caused by your code, or is it specifically because of the library?

- [x] I have double-checked my code carefully.

### Describe the bug.

```js
    window.WWebJS.getChatModel = async (chat, { isChannel = false } = {}) => {
		        if (!chat) return null;

		        const model = chat.serialize();
		        model.isGroup = false;
		        model.isMuted = chat.mute?.expiration !== 0;
		        if (isChannel) {
		            model.isChannel = window.Store.ChatGetters.getIsNewsletter(chat);
		        } else {
		            model.formattedTitle = chat.formattedTitle;
		        }

		        if (chat.groupMetadata) {
		            model.isGroup = true;
		            const chatWid = window.Store.WidFactory.createWid(chat.id._serialized);
		            await window.Store.GroupMetadata.update(chatWid); // crash here.GroupMetadata is undefined
		            chat.groupMetadata.participants._models
		                .filter(x => x.id?._serialized?.endsWith('@lid'))
		                .forEach(x => x.contact?.phoneNumber && (x.id = x.contact.phoneNumber));
		            model.groupMetadata = chat.groupMetadata.serialize();
		            model.isReadOnly = chat.groupMetadata.announce;
		        }
// ...
```

error stacks:

```
VM200:689 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'update')
    at window.WWebJS.getChatModel (<anonymous>:689:48)
    at <anonymous>:664:64
    at Array.map (<anonymous>)
    at window.WWebJS.getChats (<anonymous>:664:38)
```

### Expected Behavior

getChats should return chats

### Steps to Reproduce the Bug or Issue

call indow.WWebJS.getChats

### WhatsApp Account Type

Standard

### Browser Type

Chromium

### Operation System Type

Windows 11

### Phone OS Type

iOS

### WhatsApp-Web.js Version

1.34.5-alpha.3

### WhatsApp Web Version

2.3000.1032542123

### Node.js Version

v22.212

### Authentication Strategy

LocalAuth

### Additional Context

_No response_

i am also facing same issue.

```
[ERROR] 2026-01-28T20:11:53.874Z - Error getting groups: Error [TypeError]: Cannot read properties of undefined (reading 'update')
window.WWebJS.getChatModel (pptr:evaluate;Binding.%3Canonymous%3E%20(%2FUsers%2Fhasanmac%2FDevs%2Fcrm-project%2Fscrapbox%2Fwhatsapp-scrap%2Fnode_modules%2Fwhatsapp-web.js%2Fsrc%2FClient.js%3A265%3A36):640:46)
    at window.WWebJS.getChatModel (evaluate at Binding.<anonymous> (/Users/Devs/crm-project/whatsapp/node_modules/whatsapp-web.js/src/Client.js:265:36), <anonymous>:639:45)

```

--- a/src/util/Injected/Utils.js
+++ b/src/util/Injected/Utils.js
@@ -645,7 +645,8 @@ exports.LoadUtils = () => {
         if (chat.groupMetadata) {
             model.isGroup = true;
             const chatWid = window.Store.WidFactory.createWid(chat.id._serialized);
-            await window.Store.WAWebGroupMetadataCollection.update(chatWid);
+            const groupMetadata = window.Store.GroupMetadata || window.Store.WAWebGroupMetadataCollection;
+            await groupMetadata.update(chatWid);
             chat.groupMetadata.participants._models
                 .filter(x => x.id?._serialized?.endsWith('@lid'))
                 .forEach(x => x.contact?.phoneNumber && (x.id = x.contact.phoneNumber));
@@ -654,7 +655,8 @@ exports.LoadUtils = () => {
         }
 
         if (chat.newsletterMetadata) {
-            await window.Store.WAWebNewsletterMetadataCollection.update(chat.id);
+            const newsletterMetadata = window.Store.NewsletterMetadataCollection || window.Store.WAWebNewsletterMetadataCollection;
+            await newsletterMetadata.update(chat.id);
             model.channelMetadata = chat.newsletterMetadata.serialize();
             model.channelMetadata.createdAtTs = chat.newsletterMetadata.creationTime;
         }

--- a/src/util/Injected/Utils.js
+++ b/src/util/Injected/Utils.js
@@ -639,7 +639,7 @@ exports.LoadUtils = () => {
         if (chat.groupMetadata) {
             model.isGroup = true;
             const chatWid = window.Store.WidFactory.createWid(chat.id._serialized);
-            await window.Store.GroupMetadata.update(chatWid);
+            await window.Store.WAWebGroupMetadataCollection.update(chatWid);
             chat.groupMetadata.participants._models
                 .filter(x => x.id?._serialized?.endsWith('@lid'))
                 .forEach(x => x.contact?.phoneNumber && (x.id = x.contact.phoneNumber));
@@ -648,7 +648,7 @@ exports.LoadUtils = () => {
         }
 
         if (chat.newsletterMetadata) {
-            await window.Store.NewsletterMetadataCollection.update(chat.id);
+            await window.Store.WAWebNewsletterMetadataCollection.update(chat.id);
             model.channelMetadata = chat.newsletterMetadata.serialize();
             model.channelMetadata.createdAtTs = chat.newsletterMetadata.creationTime;
         }

